File: 4. Fourth_Semester/Computer-Science-Workshop-2(CSW-2)/05. Error Handling in Java/Q06.java
import java.util.Scanner;

public class Q06 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter a number : ");
        try {
            String input = scanner.nextLine();
            int num = Integer.parseInt(input);
            System.out.println("Entered num : "+num);
        } catch (NumberFormatException e) {
            System.err.println("NumberFormatException occurred : Invalid input !! Please enter a valid integer!");
        } finally {
            scanner.close();
        }
    }
}

/**
 * OUTPUT
 * Enter a number : A
 * NumberFormatException occurred : Invalid input !! Please enter a valid integer!
 */

File: 5. Fifth_Semester/PYTHON/Minor Assignment 2/Q06.py
num = int(input("Enter an Integer :"))

if(num < 2):
    print(f"{num} is not a Prime Number")
else:
    isPrime = True
    for i in range(2, num):
        if(num % i == 0):
            isPrime = False
            break
    
    if isPrime:
        print(f"{num} is a Prime number")
    else:
        print(f"{num} is not a Prime number")